Apps/Sandcastle/CesiumSandcastle.css

```diff
@@ -174,6 +174,17 @@ a.linkButton:focus, a.linkButton:hover {
     overflow: hidden;
 }
 
+.bottomPanel #innerPanel_tablist {
+    overflow-y: hidden!important;
+    max-height: 34px;
+}
+
+@media (max-width: 1064px){
```

Comment:
Do we want to have this be a media query, or just set a max height the height of the labels and how overflow be auto? 

I can get the same behavior with just 

```
    max-height: 28px;
    overflow: auto !important;
```